tests/test_config.py

```
@@ -137,6 +138,22 @@ def test_noenv_nodef_nosetup_pyproject(self):
             self.assertEqual(self.pyproject_context, cfg.getint("ipdb", "context"))
             self.assertRaises(configparser.NoOptionError, cfg.get, "ipdb", "version")
 
+    def test_noenv_nodef_nosetup_pyproject(self):
```

Ruff (0.15.2)

[error] Redefinition of unused `test_noenv_nodef_nosetup_pyproject` from line 126: `test_noenv_nodef_nosetup_pyproject` redefined here

(F811)